_posts/2020-04-28-mocking.adoc

```diff
@@ -0,0 +1,365 @@
+---
+layout: post
+title: 'Mocking CDI beans in Quarkus'
+date: 2020-04-28
+tags: testing
+synopsis: Quarkus 1.4 introduces some interesting new capabilities for mocking CDI beans.
+author: geoand
+---
+
+Testing Quarkus applications has been an important part of the Quarkus Developer Joy, which is why
+`@QuarkusTest` for testing JVM applications and `@NativeTest` for black-box testing of the native images have been
+part of Quarkus since the first release.
+A recurring request however amongst our community members has been to have Quarkus allow them to selectively
+mock certain CDI beans for specific tests. This post will introduce the new mocking capabilities
+that `1.4.0.Final` brings which aim to address those concerns, while also providing a glimpse of additional
+improvements in this are that will be part of `1.5.0.Final`.
```

Comment:
I would refer to 1.4 and 1.5 for the version, especially since we don't want people to use 1.4.0.Final

Reply:
Good call, fixed